crates/stackable-operator/CHANGELOG.md

```diff
@@ -4,11 +4,18 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Changed OPA Bundle Builder Vector config to read from the new log-to-file setup ([#814]).
+
+[#814]: https://github.com/stackabletech/operator-rs/pull/814
+
 ## [0.70.0] - 2024-07-10
 
 ### Added
 
 - Added `ProductImage::product_version` utility function ([#817], [#818])
+- Added support for logging to files ([#814]).
```

Comment:
This entry must also be moved to the unreleased section.

Reply:
>.< 43d3baec1c670926ca24d9bb92a24eac3f074970